packages/components/src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,194 @@
+                ref={ref}
+                className={styles.root}
+                onValueChange={(value) => {
+                    if (value) {
+                        setActiveTab(value);
+                    }
+                }}
+                value={activeTab ?? triggers[0]?.value}
+                {...props}
+            >
+                <div className={styles.triggerListWrapper}>
+                    <RadixTabs.List ref={triggerListRef} data-size={size} className={styles.triggerList}>
+                        {triggers.map((trigger) => (
+                            <RadixTabs.Trigger
+                                key={trigger.value}
+                                value={trigger.value}
+                                disabled={trigger.disabled}
+                                className={styles.trigger}
+                                asChild
+                            >
+                                {trigger.element}
+                            </RadixTabs.Trigger>
+                        ))}
+                    </RadixTabs.List>
+                    <Dropdown.Root>
+                        {triggersOutOfView.length > 0 && (
+                            <Dropdown.Trigger data-test-id="overflow-items-dropdown-trigger">
+                                <Button emphasis="default" aspect="square" size="small">
+                                    <IconDotsHorizontal size={16} />
+                                </Button>
+                            </Dropdown.Trigger>
+                        )}
+                        <Dropdown.Content align="end" data-test-id="overflow-items-dropdown-content">
+                            {triggersOutOfView.map((trigger) => (
+                                <Dropdown.Item
+                                    disabled={trigger.disabled}
+                                    onSelect={() => {
+                                        setActiveTab(trigger.value);
+                                    }}
+                                    key={trigger.value}
+                                >
+                                    {trigger.element}
+                                </Dropdown.Item>
+                            ))}
+                        </Dropdown.Content>
+                    </Dropdown.Root>
+                    <span
+                        data-test-id="active-tab-indicator"
+                        ref={activeIndicatorRef}
+                        className={styles.activeIndicator}
+                    />
+                </div>
+                {children}
+            </RadixTabs.Root>
+        </TabTriggerContext.Provider>
+    );
+};
+TabsRoot.displayName = 'Tabs.Root';
+
+type TabsTabProps = {
+    children: ReactNode;
+    value: string;
+    disabled?: boolean;
+};
+
+export const TabsTab = ({ children, value, disabled }: TabsTabProps) => (
+    <TabConfigContext.Provider value={{ value, disabled: !!disabled }}>{children}</TabConfigContext.Provider>
```

Comment:
`disabled` is already a boolean, if not possible prefer the fallback approach `disabled ?? false/true`
```suggestion
    <TabConfigContext.Provider value={{ value, disabled }}>{children}</TabConfigContext.Provider>
```

Reply:
fixed